mlir/lib/Target/LLVMIR/DebugImporter.cpp

```diff
@@ -208,6 +208,21 @@ DINamespaceAttr DebugImporter::translateImpl(llvm::DINamespace *node) {
                               node->getExportSymbols());
 }
 
+DIImportedEntityAttr
```

Comment:
This requires a clear TODO comment that we drop this information that also explains why it is dropped. Ultimately, we would want to import even the cyclic structures, but this can wait for now.

Reply:
Maybe add a brief comment on the attribute itself why there is no scope field.

Reply:
I added a comment on the attribute.